libcodechecker/generic_package_context.py

```diff
@@ -23,6 +24,36 @@
 LOG = logger.get_logger('system')
 
 
+class SeverityMap(Mapping):
```

Comment:
Couldn't we use `MutableMapping`? `__setitem__` and `__delitem__` magic functions belong to `MutableMapping`.